tests/e2e/demo_test.go

```diff
@@ -86,7 +86,7 @@ var _ = Describe("katacodaDemo", func() {
 			runCmd("odo url create frontend")
 
 			getUrl := runCmd("odo url list")
-			Expect(getUrl).To(ContainSubstring("frontend-sample-proj"))
+			Expect(getUrl).To(ContainSubstring("frontend-sample-sample-proj"))
```

Comment:
@kadel Just alerting to you that documentation may need to be changed on Katacoda for this. Since it will now be: componentName-applicationName-projectName